config/default/config.json

```diff
@@ -77,7 +77,10 @@
 		"relayLimit": 3
 	},
 	"transactions": {
-		"maxTransactionsPerQueue": 1000
+		"maxTransactionsPerQueue": 1000,
+		"expiryInterval": 30000,
+		"multiSignatureTimeoutMultiplier": 3600,
+		"signaturesTimeoutMultiplier": 8
```

Comment:
We have `unconfirmedTransactionTimeOut` and `blockReceiptTimeOut` in constants. So we need revisit the though tif these new timeout values should go to `config` or `constants`. 

General rule can be, If these values can be configured differently on each node and still node works smoothly with the network, then its a config. If each node have to have same value in network to operate then its a constants. 

@MaciejBaj can confirm where to put these values in `constants` or `config`?

Reply:
Agree, if `3600` and `8` are constant values which is specific, then we can move it to `constants`, @MaciejBaj can you please give me the feedback?